### ISSUE TITLE
Live character limit display

### ISSUE THREAD

**Issue by DNin01** [ScratchAddons/ScratchAddons#6183]
### Why this would be helpful

Some text fields around Scratch have character limits that aren't visible to the user until they try to save the content. These fields include forum posts and project descriptions, which I believe have limits of 200,000 and 5,000 characters respectively. It would be nice if you could know how close to the limits you're getting so you can make adjustments before the field gets too big.

### How this addon works

It would display a counter that tells you how many characters are left in text fields with character limits.

![Demonstration in Instructions field](https://github.com/ScratchAddons/ScratchAddons/assets/106490990/7328a51a-e3dd-4eb0-9b3b-57c0bcea71ba)

**Comment by WorldLanguages** [comment]
Good idea, we need to make sure we find out the actual character limits (and hopefully they don't vary between New Scratchers vs full Scratchers?)

**Comment by DNin01** [comment]
For the project descriptions, [I can confirm the limit is 5,000 characters](https://github.com/scratchfoundation/scratch-www/blob/3970ff45147c8c8c0628211045bc4899f6894bfc/src/views/preview/presentation.jsx#L461). (I even checked it in the browser developer tools.)

**Comment by mxmou** [comment]
I'm not sure if this makes sense for forum posts - the only was you would ever reach 200,000 characters is by copying and pasting huge amounts of data from somewhere.

**Comment by CST1229** [comment]
> I'm not sure if this makes sense for forum posts - the only was you would ever reach 200,000 characters is by copying and pasting huge amounts of data from somewhere.
> 
> I think signatures have a shorter limit though.

I think signatures have a 2000 character limit? (I don't remember though.)

**Comment by mybearworld** [comment]
![An image showing that the signature character limit is 2,000.](https://github.com/ScratchAddons/ScratchAddons/assets/130385691/ffe9977b-1e2d-4f76-bfc9-c53374d43166)

**Comment by DNin01** [comment]
Self-assigning because I'm working on this.
